source/screens/caseScreens/CaseSummary.js

```diff
@@ -200,33 +224,49 @@ const computeCaseCardComponent = (
   );
 };
 
+async function updateCaseSignature(caseItem, signatureSuccesful) {
+  const currentForm = caseItem.forms[caseItem.currentFormId];
+
+  const updateCaseRequestBody = {
+    answers: currentForm.answers,
+    currentPosition: currentForm.currentPosition,
+    currentFormId: currentForm.id,
+    signature: { success: signatureSuccesful },
+  };
+  try {
+    const updateCaseResponse = await put(`/cases/${caseId}`, JSON.stringify(updateCaseRequestBody));
```

Comment:
"put" missing import statement.
"caseId" not defined, should be "currentForm.id"?